packages/next/build/index.ts

```diff
@@ -361,6 +363,10 @@ export default async function build(dir: string, conf = null): Promise<void> {
           })
           staticCheckSema.release()
 
+          if (result.prerender === true) {
```

Comment:
This should come after the static check -- static pages don't need to be prerendered.
```js
          if (result.static && customAppGetInitialProps === false) {
            staticPages.add(page)
            isStatic = true
          } else if (result.prerender === true) {
            sprPages.add(page)
          }
```

Reply:
Good catch